doc/nondeterministic.md

```diff
@@ -68,18 +64,17 @@ class MyTests : ShouldSpec() {
 }
 ```
 
-**ATTENTION**: `eventually` does not works well with `assertSoftly`. In `assertSoftly` if an `eventually` fails any
-assertion followed by it won't be executed. Example:
+**ATTENTION**: `eventually` currently does not works well within `assertSoftly`, since assertions aren't thrown normally in there:
```

Comment:
since you fixed the "s" on the comment bellow I figure you don't want the s on the "works"